src/test/java/seedu/address/storage/JsonTeamListStorageTest.java

```diff
@@ -61,25 +61,48 @@ public void readTeamList_invalidAndValidTeamList_throwDataConversionException()
         assertThrows(DataConversionException.class, () -> readTeamList("invalidAndValidTeamList.json"));
     }
 
-    @org.junit.jupiter.api.Test
+    @Test
     void readAndSaveTeamList_allInOrder_success() throws Exception {
+        saveAndCompareTeamList(TypicalTeams.getTypicalTeamList(),
+                               TypicalTeams.getTypicalTeams());
```

Comment:
Any reason why you chose to do the assert in the helper function instead of in the test function directly?

Reply:
This is to abstract away the details of the assertions, so that I can simply call a single function from all 3 readAndSaveTeamList tests